data/model/oci/test/test_oci_manifest.py

```diff
@@ -73,7 +73,7 @@ def create_manifest_for_testing(repository, differentiation_field="1"):
 
     remote_digest = sha256_digest("something")
     builder = DockerSchema2ManifestBuilder()
-    builder.set_config_digest(config_digest, len(layer_json))
+    builder.set_config_digest(config_digest, len(layer_json.encode("utf-8")))
```

Comment:
@kleesc 
Was this `layer_json` serialized by Python? I'm curious why it has to be encoded if it was created in Python which should automatically generate utf-8 instead of bytes/ASCII

Reply:
json_dumps() returns a str and not a bytes. It's encoded here to get the correct length if there are non-ascii characters, which is allowed in JSON.